corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -60,30 +73,62 @@
     });
   </script>
 
-  <form id="loc_form" class="form form-horizontal" name="product" method="post">
-    {% bootstrap_form_errors form %}
-    {% bootstrap_fieldset form "Location Information" %}
-
-    {% crispy form.custom_data.form %}
-
-    {% if consumption %}
-      <legend>{% trans "Default monthly consumption values" %}</legend>
-      {% for code, amount in consumption %}
-        <div class="control-group">
-          <label class="control-label">{{ code }}</label>
-          <div class="controls">
-            <span class="uneditable-input">
-              {{ amount }}
-            </span>
-          </div>
-        </div>
-      {% endfor %}
+  <ul class="nav nav-tabs">
```

Comment:
Added tabs to the location edit page, currently just "basic" and "products", but I think there's another in the works.